### PR TITLE
PMM-7883 Add PMM single line installer.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-7883

`curl -fsSL https://raw.githubusercontent.com/percona/pmm/PMM-7883-single-line-installer/get-pmm.sh  -o get-pmm2.sh ; chmod +x get-pmm2.sh ; ./get-pmm2.sh  `

### CODE REVIEW

File: get-pmm.sh [hunk 1]
```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env bash
+
+# ###############################
+# Script to run PMM 2 
+# curl -fsSLhttps://github.com/percona/pmm/blob/PMM-2/get-pmm.sh -o get-pmm2.sh ; chmod +x get-pmm2.sh ; ./get-pmm2.sh
```
Comment (rnovikovP) [inline get-pmm.sh:5]:
```suggestion
# curl -fsSLhttps://github.com/percona/pmm/blob/PMM-2.0/get-pmm.sh -o get-pmm2.sh ; chmod +x get-pmm2.sh ; ./get-pmm2.sh
```

? I assume the branch will be the same as we have now and it's PMM-2.0 for now

**Review by percona-csalguero — DISMISSED** [review]

**Review by percona-csalguero — DISMISSED** [review]

**Review by BupycHuk — DISMISSED** [review]

File: get-pmm.sh [hunk 2]
```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env bash
+#
+# ###############################
+# Script to run PMM 2.
+# If docker is not installed - this script try to install it as root user.
```
Comment (atymchuk) [inline get-pmm.sh:5]:
```suggestion
# If docker is not installed, this script will try to install it as root user.
```

File: get-pmm.sh [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+#######################################
+show_message() {
+  msg "PMM Server has been successfully setup on this system!"
+  msg "You can access your new server using the one of following web addresses:"
+  for ip in $(ifconfig | grep "inet " | awk '{print $2}'); do
+    msg "\t${BLUE}https://$ip:$port/${NOFORMAT}"
+  done
+  msg "The default username is '${PURPLE}admin${NOFORMAT}' and password is '${PURPLE}admin${NOFORMAT}'"
+  msg "**Note** Browser may not trust the default SSL certificate on first load."
+  msg "So type '${PURPLE}thisisunsafe${NOFORMAT}' to bypass their warning"
```
Comment (atymchuk) [inline get-pmm.sh:229]: This is Chrome only, Firefox won't have it. So need to mention it somehow.

Comment (atymchuk) [inline get-pmm.sh:229]:
```suggestion
  msg "So type '${PURPLE}thisisunsafe${NOFORMAT}' to bypass their warning (Chrome only)"
```

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Review by atymchuk — DISMISSED** [review]

**Review by percona-csalguero — DISMISSED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**